docs/gateway-configuration/network-failover.md

```diff
@@ -0,0 +1,33 @@
+# Network Failover
+
```

Comment:
"and a network failover basic functionality."

and basic network failover functionality? maybe this is better

Reply:
Yes, it is better